src/shared.ts

```diff
@@ -11,6 +11,7 @@ export interface PartnerParams {
   user_id: string | number;
   job_id: string | number;
   job_type: number;
+  [key: string]: unknown
```

Comment:
do we only want to support strings? Is there an associated spec change?

Reply:
Will add an associated spec

Reply:
waiting on the spec to approve this.